src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
    private ColorSupplier colorSupplier = new CS();
```
create a field and call method on this field object